.github/workflows/rust.yml

```diff
@@ -38,17 +39,19 @@ jobs:
       id: clippy
       run: .github/workflows/clippy.sh ${{ matrix.rust-toolchain }}
       continue-on-error: ${{ matrix.rust-toolchain != '1.66' }}
-    - name: Notify clippy failure
-      if: ${{ (steps.clippy.outcome != steps.clippy.conclusion) && (matrix.rust-toolchain != '1.66') }}
-      uses: actions/github-script@v6
+    - name: Store clippy flag
```

Comment:
I believe the nightly tests need the same behavior as this?

Reply:
This will run with the nightly tests in the matric unless you are referring to something else.

Reply:
I mean the "notify nightly test failure" step on line 89.